subsys/ipc/ipc_service/backends/ipc_rpmsg_static_vrings.c

```diff
@@ -250,10 +250,34 @@ static int vr_shm_configure(struct ipc_static_vrings *vr, const struct backend_c
 		return -ENOMEM;
 	}
 
+	/*
+	 * conf->shm_addr  +--------------+  vr->status_reg_addr
+	 *		   |    STATUS    |
+	 *		   +--------------+  vr->shm_addr
+	 *		   |              |
+	 *		   |              |
+	 *		   |   RX BUFS    |
```

Comment:
I thought RX_BUFS boundaries should be cache-line-aligned as well.

Looking through the code it looks like an imprecise diagram. Buffers seem to be aligned.

Reply:
Buffers size is defined using DT so the user must take care of that really (I tried to suggest that by adding a comment to the DT property).